packages/access-api/src/config.js

```diff
@@ -136,9 +138,8 @@ export function configureSigner(config) {
  * @returns {object is `did:${string}:${string}`}
  */
 function isDID(object) {
-  if (typeof object !== 'string') return false
-  const parts = object.split(':')
-  if (parts.length <= 2) return false
-  if (parts[0] !== 'did') return false
-  return true
+  try {
+    return Boolean(DID.match({}).from(object))
```

Comment:
What’s the point of catching here to then throw ? You can simply just do .from instead of whole isDID thing.

Alternatively there’s also .is method which you could instead which will return boolean

Reply:
👍 https://github.com/web3-storage/w3protocol/pull/275/commits/5184a554ee80f29ddb74ddd9c78e3b883c4c8119